### DISCUSSION TITLE
Update custom attribute on Account model

### DISCUSSION THREAD

**Discussion by crespire** [janko/rodauth-rails#188] [Q&A] [answered]
I have a custom `name` attribute, and I am not sure how to implement the ability for users to edit that attribute. I added the extra field like the sample app no problem, but running into some trouble trying to provide an option for users to update it.

The `rodauth` object does not expose a route `change_name_path` helper for me to use, and the `Account` model doesn't have the usual paths through Rails either.

**Comment by janko** [comment, marked as answer]
If you need a form to only update that attribute, then you'd do this just like with any other model. You could implement a `PUT /account` route that accepts only the `account[name]` parameter and updates the current account. The `name` attribute is not authentication-related, so there is no point going through Rodauth.
